packages/ckeditor5-dev-env/lib/release-tools/utils/getnewreleasetype.js

```diff
@@ -14,7 +15,7 @@ const { commitTypes, transform } = require( '../changelog/writer-options' );
  */
 module.exports = function getNewReleaseType() {
 	const conventionalRecommendedBump = require( 'conventional-recommended-bump' );
-	const parserOpts = require( '../changelog/parser-options' );
+	const parserOpts = require( './parseroptions' );
```

Comment:
Ow. Now I see, that there is such a convention with `opts` across the committed files.